app/jobs/subscriptions/organization_billing_job.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module Subscriptions
+  class OrganizationBillingJob < ApplicationJob
+    queue_as :default
```

Comment:
is `default` the right queue for this job? 🤔

Reply:
@nudded 

Previously, the logic identifying the subscription to bill was done in the `clock` queue (`Clock::SubscriptionsBillerJob` was performing the `Subscriptions::BillingService.call` dirrectly). For now it only enqueues a job per organization in the `default` queue.
I don't want to add it in the `billing` queue as it has to have a higher priority that the `BillSubscriptionJob` to make sure it will not be stuck for hours every 1st days of the month

Do you think it should be pushed to the `high_priority` queue?, or to a new one (`medium_high_priority`)?

Reply:
hmmm, perhaps these new organization jobs could also be clock jobs? They are run every hour right? 🤔 
(that would put them on clock or clock_worker queue).

To me the enqueuing of these jobs feels important, I don't want them to be stuck anywhere, so I'm inclined to even consider high_priority for them? (e.g. this one should be done fast and quick, the billing can be done asynchronous on the billing queue)

Reply:
I'm not against high priority as I agree, it has to be performed quickly